### DISCUSSION TITLE
ResourceTreeEditorWidget Minimal Example

### DISCUSSION THREAD

**Discussion by shortwavedave** [eclipse-emfcloud/emfcloud#207] [Theia Tree Editor]
Hello, does anyone have a minimal example that utilizes the [ResourceTreeEditorWidget](https://github.com/eneufeld/theia-tree-editor/blob/master/theia-tree-editor/src/browser/resource/resource-tree-editor-widget.ts)?

Are you supposed to bind this in the frontend module with a uri in the NavigatableTreeEditorOptions? Is it possible to load URIs dynamically?

**Comment by ndoschek** [comment]
Hi @shortwavedave,

The official `theia-tree-editor` implementation can be found in the `eclipse-emfcloud` organzation: https://github.com/eclipse-emfcloud/theia-tree-editor

Also, you can have a look at our implementation of the `theia-tree-editor` in the [`coffee-editor`](https://github.com/eclipse-emfcloud/coffee-editor/tree/master/client/coffee-editor-extension/src/browser).
In this case, the [`CoffeeTreeEditorWidget`](https://github.com/eclipse-emfcloud/coffee-editor/blob/master/client/coffee-editor-extension/src/browser/coffee-tree/coffee-tree-editor-widget.tsx) is used to open a combination of tree- and form-based editor for a certain resource.

HTH and BR,
Nina


PS: You can also test the coffee-editor online via: https://try.theia-cloud.io/?appdef=coffee-editor

**Comment by eneufeld** [comment]
Please reopen if issue still persists
